README.md

```diff
@@ -2,66 +2,70 @@
+We use Docker to be able to run the code on *any* PC that has the Docker installed. This first part is the basics of how to run a docker container on your PC. For developing, skip to the [next part](https://github.com/vortexntnu/Vortex-ASV/tree/improvement/update_readme#docker-developer-guide)
 
+To run a service, navigate to Vortex-ASV where docker-compose.yml is located and run
 ```
 sudo docker-compose up -d <service_name>
 ```
-
-Running without a service name runs every container listed in the docker-compose file. Available services are:
-vortex - for running on hardware
-vortex_asv_sim - for running on simulator
-The -d flag (detach) runs the container in the background.
-
-To open the terminal inside the container, run
-
+Where <service name> is a name from the list:
 ```
-sudo docker-compose exec <service_name> /bin/bash
+asv
+lidar
+simulator
 ```
+The -d flag (detatch) runs the services in the background instead of taking up your terminal.
+These are defined in [docker-compose.yml](docker-compose.yml). Alternatively you could run all services simultaneously. The services will run in different containers, but because of how they are defined they share their output between containers and the host. This means a ROS instance in any container can communicate with a ROS instance on the host or another container. Note that the first time you run a service it might take a few minutes.
 
-
-In the case that you get the error message "bash: /home/vortex/asv_ws/devel/setup.bash: No such file or directory" when you enter the container terminal it means you have not built the ROS workspace. 
-
-Build the ROS workspace: 
+Running the services in this way launches the programs associated with the services. For instance, running the "simulator" service launches the gazebo simulator with the GUI disabled. This makes it so any PC can run the simulator. Some PCs could not run the simulator, but with the GUI disabled, it should run fine. To get some visualization, use rviz:
 
 ```
-cd ~/asv_ws && catkin build
+rosrun rviz rviz -d sim_config.rviz
 ```
 
-Now you can source by running
+When you are finished using the container, run
 
 ```
-source ~/asv_ws/devel/setup.bash
+sudo docker-compose down <service_name>
 ```
+# Docker, developer guide
 
-or alternatively, it will source automatically the next time you run the container. To exit the container, run
+If you are altering the code, running the services listed above is cumbersome. The way you would run the code after making new changes new changes would be:
+* Push your code
+* The image is then rebuilt on a git cloud
+* Run the container
+Rather than going through this, we can used a feature in docker called volumes. A volume is a folder which is linked between the container and the host the container the running on.  /home/vortex/asv_ws/ is a volume, meaning making any changes in the volume from the container also changes the asv_ws folder in the host and vice versa.
```

Comment:
I'll probably rewrite this section to specify a few things, but this is good for now

Reply:
Great!